README.md

```diff
@@ -164,21 +164,6 @@ Share ideas on how to improve this project.
 
```

Comment:
- [OPTIONAL] Kindly remove the Microverse logo from your README file instead you can replace it with yours if you have one 👍🏻 

- [OPTIONAL] I would also recommend adding the issue link in the contributing sections, Actually, the issue link goes to the twitter.com/ Please correct this : 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```